src/shared/components/Button/Button.tsx

```diff
@@ -30,13 +30,16 @@ const ButtonComponent: React.ForwardRefRenderFunction<HTMLButtonElement, ButtonP
   },
   ref
 ) => {
-  const clickable = !!onClick
+  const clickable = !!onClick || !!to
   const hasText = !!children
 
+  const isLinkExternal = to.includes('http')
   return (
     <StyledButton
-      as={to ? Link : undefined}
+      as={to ? (isLinkExternal ? 'a' : Link) : undefined}
       to={to}
+      target={isLinkExternal ? '_blank' : undefined}
+      href={to}
```

Comment:
Every button now has the following attributes in DOM:
<img width="181" alt="Screenshot 2021-03-30 at 15 05 34" src="https://user-images.githubusercontent.com/12646744/112993380-724a5600-9169-11eb-8cc5-3c63c9747f5e.png">

I think we could do a better job filtering those out